content/rs/references/client_references/client_php.md

```diff
@@ -59,9 +63,11 @@ var_dump($value);
 
 Example output:
 
-    $ php predis_example.php
-    Connected to Redis
-    string(3) "bar"
+```console
+$ php predis_example.php
+Connected to Redis
+string(3) "bar"
+```
 
 ## Persistent connections
```

Comment:
For the example code under Persistent connections, you might want to remove one level of indentation from the code block to make the text align left.

Reply:
Fixed.